sql/moz-fx-data-shared-prod/firefox_ios_derived/profile_dau_metrics_marketing_geo_testing_v1/query.sql

```diff
@@ -0,0 +1,46 @@
+SELECT
+  active_users.submission_date,
+  active_users.normalized_channel,
+  active_users.app_name,
+  active_users.app_display_version AS app_version,
+  COALESCE(active_users.country, '??') AS country,
+  active_users.city,
+  active_users.geo_subdivision,
+  active_users.locale,
+  active_users.os,
+  active_users.os_version,
+  active_users.device_manufacturer,
+  active_users.device_type,
+  active_users.is_mobile,
+  active_users.is_suspicious_device_client,
```

Comment:
this field doesn't seem to exist in `moz-fx-data-shared-prod.firefox_ios.active_users`. Maybe it's in a view or something?